app/views/profiles/edit.html.erb

```diff
@@ -40,6 +40,21 @@
     </div>
   </fieldset>
 
+  <fieldset class="mb-3">
+    <legend><%= t ".social_links.legend" %></legend>
+    <div id="social_links">
+      <%= f.fields_for :social_links do |social_link_form| %>
+        <div class="social-link-fields row mb-3">
+          <%= social_link_form.text_field :url, :hide_label => true, :wrapper_class => "col-sm-8" %>
+          <%= social_link_form.check_box :_destroy, :wrapper_class => "d-none social_link_destroy" %>
+          <%= social_link_form.label :_destroy, t(".social_links.remove"), :class => "btn btn-outline-primary col-sm-1 align-self-start" %>
+        </div>
+      <% end %>
+    </div>
+
+    <%= link_to "+", "#", :id => "add-social-link", :class => "btn btn-outline-primary" %>
```

Comment:
Why is it a link and not a button?

Why adding is "+" but removing is "Remove"?

Reply:
I had a different logic that needed a link, as I changed logic, I agree with you, I'll change it to button.

My idea was that `+` sign doesn't require any additional icons and is intuitive, while `Remove` needs `trash icon` to be intuitive (`-` sign doesn't always mean removal and sometimes it may be ambiguous). But I agree with you, maybe it's better to be consistent in this too. Which one do you think is better, to change `+` sign to `Add` (which will be much easier and less-code solution) or to change `Remove` to `trash icon` (which may be nicer in terms of UI, but will require trash icon and, therefore, more code)?